omnibus/config/projects/agent.rb

```diff
@@ -110,6 +110,9 @@
   skip_healthcheck true
 else
   do_build = true
+  if ENV["OMNIBUS_FORCE_PACKAGES"]
+    do_package = true
+  end
```

Comment:
Do we need to whitelist this variable somewhere in the python task?

Reply:
That depends on which list you're thinking about.
We could ignore it for the omnibus cache key generation (but since it's meant for local builds only I don't think we really care)
We might need to ensure it's forwarded to the omnibus env, but during my testing this was not required (but I thought it would, so I'll take a closer look)

If you were thinking about another list, let's discuss it :)

Reply:
> We might need to ensure it's forwarded to the omnibus env, but during my testing this was not required (but I thought it would, so I'll take a closer look)

I thank thinking about this one

Reply:
Oh well, I thought the environment we use in the omnibus build tasks was fully provided by the env parameter, but I was wrong.
Whatever we provide to `omnibus_run_task` (or to `invoke.runner.Run` for that matter) as the `env` parameter will modify the current process environment.
Put another way I thought we were using `{}.update(env)` but we're using `os.environ.update(env)`

Reply:
I'll add the variable to the explicitely forwarded environment in case this behavior changes when we fully switch to deva, of we find a way to enforce a cleaner env though invoke